src/mrack/data/provisioning-config.yaml

```diff
@@ -21,17 +24,7 @@ podman:
         "--cap-add":
             - "ALL"
         # Allowed syscall list seccomp JSON file to be used as a seccomp filter
-        "--security-opt": "seccomp=src/mrack/data/seccomp.json"
-        # Mount a temporary filesystems (tmpfs) into a container
-        "--tmpfs":
-            - "/tmp"
-            - "/run"
-            - "/run/lock"
-        # Use /sys/fs/cgroup in container as read only volume
-        "-v":
-            - "/sys/fs/cgroup:/sys/fs/cgroup:ro"
-        # Adding ipv6 support to network
-        "--network": "enable_ipv6=true"
```

Comment:
This file is just a means for documentation and serves as an example. It doesn't affect execution. I would probably only remove the last line